examples/ExampleLoginPlugin.jsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
+import { Component } from "react";
+
+class ExampleLoginPlugin extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    const { onPermissionOverrides } = this.props;
+    const permissionOverrides = {
+      canUsePanels: true,
+      canCreateDashboard: true,
+      canCreateCodeStudio: true,
+      canCreateQueryMonitor: true,
+      canCopy: true,
+      canDownloadCsv: true,
+    };
+    onPermissionOverrides((username, operateAs, groups) => {
+      console.log("[ExampleLoginPlugin]", username, operateAs, groups);
+      return permissionOverrides;
+    });
```

Comment:
I think put the object right in line, no need to create it before:
```suggestion
    const { onPermissionOverrides } = this.props;
    onPermissionOverrides((username, operateAs, groups) => {
      console.log("[ExampleLoginPlugin]", username, operateAs, groups);
      return {
        canUsePanels: true,
        canCreateDashboard: true,
        canCreateCodeStudio: true,
        canCreateQueryMonitor: true,
        canCopy: true,
        canDownloadCsv: true,
      };
    });
```

Reply:
Fixed